File: modul-02/python/PRAK204-2310817310003-Mfaridsetiawan.py
nilai = input().split()
if len(nilai) == 1:
    r = float(nilai[0])
    t = float(input())
elif len(nilai) == 2:
    r,t = map(float, nilai)

phi = 3.142857

volume = phi * r ** 2 * t
luas = 2 * phi * r * (r + t)
keliling = 2 * phi * r

print(f"Alas = {volume:.2f}")
print(f"Tinggi = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

File: modul-02/bahasa c/PRAK204-2310817310003-Mfaridsetiawan.c
#include <stdio.h>
int main(){
    float phi = 3.142857;
    float r, t;
scanf("%f %f", &r, &t);
printf("Volume : %.2f \n", phi *r * r * t);
printf("Luas : %.2f \n",2 * phi * r * (r + t));
printf("Keliling : %.2f \n", 2 * phi * r);

return 0;

}